webserver/src/main/java/io/kestra/webserver/services/ai/gemini/GeminiAiService.java

```diff
@@ -0,0 +1,362 @@
+                    - Beware about `default` values of properties and if needed tweak them to avoid too long waiting times or if a better value could be used to suit user's intent.
+                    - Root-level outputs in the Flow are used to return values from the Flow execution, not display them. If the user asks explicitly that the Flow outputs something for another flow / execution to use it, you can include some, otherwise it's useless.
+                    - If a loop through index is needed (for example to do something X times), use ForEach task. If looping until a condition is met, use LoopUntil task instead.
+                    - Detect data fetching requirements. If the initial Flow doesn't provide data that the user wants to use or display, make sure to fetch it in a prior task (HTTP, database, ...).
+                    - If there is a trigger concept and a concept of "new" most likely you'll need to leverage KV tasks to fetch a state and store it at the end so that you detect changes between executions.
+                    - Triggers and tasks have very different purposes. Triggers are used to start a Flow execution based on an event or an interval, while tasks are used to perform actions within the Flow. Make sure to distinguish between them and include the needed ones.
+                    - If there is a concept of triggering an execution based on an event or on an interval or listening for an event, make sure to include AT LEAST 1 trigger.
+                    - Unless specified by the user, never assume a local port to serve any content, always use a remote URL (like a public HTTP server) to fetch content.
+                    - Unless specified by the user, do not use any authenticated API, always use public APIs or those that don't require authentication.
+                    - To avoid escaping quotes, use double quotes first and if you need quotes inside, use single ones. Only escape them if you have 3+ level quotes, for example: `message: "Hello {{inputs.userJson | jq('.name')}}"`.
+                    - A property key is unique within each type.
+                    - If root-level `id` / `namespace` properties are provided, you MUST keep them as-is in the Flow YAML.
+                    - If a mention of "it" or "that" or similar is made, you MUST deduce the subject from the current Flow YAML (example: if the user says "display it", and the current Flow YAML fetches the weather, you should display the weather data).
+                    - IMPORTANT: If the user prompt cannot be fulfilled with the schema, instead of generating a Flow, reply: "%s".
+                    - IMPORTANT: Beside of the two above scenario which don't generate YAML but an error message, output only the raw YAML, nothing else.
```

Comment:
```suggestion
                    - Except for error scenarios, output only the raw YAML, with no explanation or additional text.
```

Don't mark 3 consecutive sections as IMPORTANT 😄 it's like highlighting an entire page with a highlighter. if everything is marked as important, nothing ist.